tests/acceptance/administrator/UserCest.php

```diff
@@ -48,4 +47,95 @@ public function administratorCreateUser(\AcceptanceTester $I)
 		$I->expectTo('see a success message and the user added after saving the user');
 		$I->see('User successfully saved', ['id' => 'system-message-container']);
 	}
-}
\ No newline at end of file
+
+	public function administratorEditUser(\AcceptanceTester $I)
+	{
+		$I->am('Administrator');
+		$I->wantToTest('Editing a user');
+
+		$I->doAdministratorLogin();
+		$I->amGoingTo('Navigate to Users page in /administrator/');
+		$I->amOnPage('administrator/index.php?option=com_users&view=users');
+
+		$I->expectTo('see Users page');
+		$I->checkForPhpNoticesOrWarnings();
+		$I->click($this->name);
+		$I->checkForPhpNoticesOrWarnings();
+		$I->waitForText('Users: Edit', '30', ['css' => 'h1']);
```

Comment:
I would like to give two suggestions here,
1. Instead of css , the class locater will perform faster. You should try: ['class' => 'page-title']
2. I think instead of '30' it should be 30 as it is timeout and should be integer.